### DISCUSSION TITLE
Strange behavior when trying to get ctx response info

### DISCUSSION THREAD

**Discussion by kskrlinnorth2** [strapi/strapi#6862] [Legacy Topics]
**Describe the bug**
```
{
    "response": {
        "status": 200,
        "message": "OK",
        "header": {
            "vary": "Origin",
            "strict-transport-security": "max-age=31536000; includeSubDomains",
            "x-frame-options": "SAMEORIGIN",
            "content-type": "application/json; charset=utf-8",
            "x-powered-by": "Strapi <strapi.io>",
            "x-response-time": "3ms"
        }
    },
    "status": 404,
    "message": "Not Found",
    "body": {
        "email": "email value"
    }
}
```
This is full response from custom POST method.
`response` is equal to `ctx.response`, `status` is equal to `ctx.response.status`, `message` is equal to `ctx.response.message`, while `body` is equal to `ctx.request.body`

**Expected behavior**
I expect to get 200 and message "Ok", same as in ctx.response. Why values for status and message are different when reading from that same object?

**System**

- Node.js version: 12.13
- NPM version: 6.12
- Strapi version: latest
- Database: not important for this since there is no communication with db at this point
- Operating system: W10

**Additional context**
Postman used for testing response data.
Custom controller and route.

```
module.exports = {
  send: async (ctx) => {
    const response = ctx.response,
      body = ctx.request.body;

    return {
      response: response,
      status: response.status,
      message: response.message,
      body: body,
    };
  },
};
```

```
{
  "routes": [
    {
      "method": "POST",
      "path": "/contact",
      "handler": "contact.send"
    }
  ]
}
```

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

**Comment by derrickmehaffy** [comment]
@alexandrebodin @lauriejim can you add anything here?

**Comment by kskrlinnorth2** [comment]
How it is not bug if it shows wrong values?
